### DISCUSSION TITLE
Critical error enabling high-performance order storage

### DISCUSSION THREAD

**Discussion by mateusscardosoo** [woocommerce/woocommerce#43133] [WooCommerce HPOS] [answered]
Hey guys!

I activated the High performance Order Storage function in my official environment, and it gave a critical error.

I can’t deactivate it, as the hosting admin panel is inaccessible when there is an error. The short-term solution I had was to deactivate the woocoomerce plugin through hosting and return it to version 7.9.0.

Do I need some way to disable this function to help me?

Thank you very much in advance!

**Comment by lsinger** [comment, marked as answer]
Hey @mateusscardosoo,

Thanks for opening up the discussion, but this sounds like it might be a better fit for [an issue](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=&projects=&template=1-bug-report.yml). The template for creating a new issue also contains field for entering your system status report. That and any additional details about your environment will help us understand what's going on. E.g., please also include any error messages you're seeing. 

Note that if you have access to the `wp` CLI tool, you can also use that to disable HPOS: 

`wp wc cot disable`
